tests/baselines/reference/inferPartialTypeArgumentsErrors.errors.txt

```diff
@@ -0,0 +1,108 @@
+tests/cases/conformance/types/typeParameters/typeParameterLists/inferPartialTypeArgumentsErrors.ts(5,50): error TS2322: Type 'string' is not assignable to type 'B'.
+  'string' is assignable to the constraint of type 'B', but 'B' could be instantiated with a different subtype of constraint 'string'.
+tests/cases/conformance/types/typeParameters/typeParameterLists/inferPartialTypeArgumentsErrors.ts(5,55): error TS2322: Type 'string' is not assignable to type 'B'.
+  'string' is assignable to the constraint of type 'B', but 'B' could be instantiated with a different subtype of constraint 'string'.
+tests/cases/conformance/types/typeParameters/typeParameterLists/inferPartialTypeArgumentsErrors.ts(11,41): error TS2344: Type 'string' does not satisfy the constraint '?'.

```

Comment:
The error reporting here definitely could be improved, the `?` marker should not be printed here. I can take care of that after getting some initial feedback.